packages/caravan-fees/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@caravan/fees",
+  "version": "1.0.0-beta",
+  "description": "Utility library for fee bumping bitcoin transactions",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/index.mjs",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
+  },
+  "scripts": {
+    "build": "tsup src/index.ts --format cjs,esm --dts",
+    "ci": "npm run lint && npm run test",
+    "dev": "npm run build -- --watch",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "test": "jest src/tests",
+    "test:watch": "jest --watch src",
+    "lint": "eslint src"
+  },
+  "keywords": [
+    "bitcoin",
+    "cpfp",
+    "rbf",
+    "feebumping",
+    "blockchain"
+  ],
+  "author": "Mrigesh Thakur",
+  "license": "MIT",
+  "engines": {
+    "node": ">=20"
+  },
+  "devDependencies": {
+    "prettier": "^3.2.5",
+    "@inrupt/jest-jsdom-polyfills": "^3.2.1",
```

Comment:
do you know why this was necessary?

Reply:
It was added to resolve the issue I was facing with getting jest to run tests ...
<img width="498" alt="Screenshot 2024-09-28 at 03 50 02" src="https://github.com/user-attachments/assets/a70123fc-c7db-4093-bede-47a5dd33957a">


import { TextEncoder, TextDecoder } from "util";
import "@inrupt/jest-jsdom-polyfills";

// Polyfill TextEncoder and TextDecoder
global.TextEncoder = TextEncoder as any;
global.TextDecoder = TextDecoder as any;

// Define `self` for environments where it's not available (like Node.js)
(global as any).self = global;

// Polyfill missing properties for `Window` type compatibility
(global as any).name = "";


this made the things working ...

Reply:
Ah, right. Thanks! I remember now 😄